### PR TITLE
Course page

### PR DESCRIPTION
**Pull request by ryanmadamba**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -158,15 +159,11 @@ function App() {
   }, []);
 
   return (
-    <div>
-
```
Comment (tddumont) [inline src/App.js:162]: Thanks for cleaning up the spacing!

File: src/Components/CourseScreen.css [hunk 2]
```diff
@@ -94,25 +91,7 @@ p {
   text-align: center;
 }
 
-/* button:hover {
```
Comment (tddumont) [inline src/Components/CourseScreen.css:97]: Good to remove unused code.

File: src/Components/Courses.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+  const courses =
+  {
+    "courseInfos": [
+      {
+        "courseId": "e510f27e125",
+        "course": "English",
+        "instructor": "Mr. Butcher",
+        "description": "Shakespeare and other classics"
+      },
```
Comment (samiur98) [inline src/Components/Courses.js:21]: Looks good.

File: src/Components/Courses.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+      //if (courseItem.course === 'Integrated Science') {
+        return ( 
+          <div key={courseItem.courseId} className='courseItem' onClick={() => changeToMissionsScreen(courseItem.courseId)}>
+            <ul>
+              {courseItem.course}
+            </ul>
+          </div>
+        )
+      } 
+      /*else {
```
Comment (tddumont) [inline src/Components/Courses.js:80]: Maybe consider removing this commented code, if it is not going to be used.

File: src/gqlQueries.js [hunk 5]
```diff
@@ -119,6 +119,19 @@ export const GET_MISSION = gql`
   }
 `;
 
+export const GET_COURSE = gql`
+  query getCourse($id: String!){
+    courseInfo(courseId: $id){
+      courseId
+      course
+      instructor
```
Comment (samiur98) [inline src/gqlQueries.js:127]: Looks Good.

**Review by samiur98 — APPROVED** [review]
Looks Good.

**Review by tddumont — APPROVED** [review]
New code looks great! Also, thanks for getting rid of unnecessary line-breaks and commented code!

**Pull request merged**